### PR TITLE
Update gdpr-dsr-Dynamics365.md

### PR DESCRIPTION
**Pull request by shellyhaverkamp**
Hello! I made some updates to the section on Dynamics 365 Customer Service Insights. The links point to topics that will be updated and published live on 4/30/2019 to coincide with the April '19 release.

Please send a review link before merging/publishing to live. Thank you!

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/gdpr-dsr-Dynamics365.md [hunk 1]
```diff
@@ -97,7 +97,17 @@ In Dynamics 365 for Marketing, you have the following additional capabilities:
 
 2.  Utilize the Insight Views on contacts and objects of marketing execution to identify additional data points that may contain customer data.
 
-Dynamics 365 Customer Service Insights provides capabilities for you to respond to requests for copies of personal data, to delete personal data, and to find the personal data that is the subject of Data Subject Discovery Requests (DSRs). For details, see [Overview of GDPR compliance for Dynamics 365 Customer Service Insights](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-summary).
+Dynamics 365 Customer Service Insights provides these capabilities: 
+
+1. [Find Customer Service Insights resources containing customer data](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-discovery) in order to respond to GDPR requests from customers.
```
Comment (nenonix) [inline microsoft-365/compliance/gdpr-dsr-Dynamics365.md]:
```suggestion
1. [Find Customer Service Insights resources containing customer data](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-discovery) to respond to GDPR requests from customers. 
```

File: microsoft-365/compliance/gdpr-dsr-Dynamics365.md [hunk 2]
```diff
@@ -97,7 +97,17 @@ In Dynamics 365 for Marketing, you have the following additional capabilities:
 
 2.  Utilize the Insight Views on contacts and objects of marketing execution to identify additional data points that may contain customer data.
 
-Dynamics 365 Customer Service Insights provides capabilities for you to respond to requests for copies of personal data, to delete personal data, and to find the personal data that is the subject of Data Subject Discovery Requests (DSRs). For details, see [Overview of GDPR compliance for Dynamics 365 Customer Service Insights](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-summary).
+Dynamics 365 Customer Service Insights provides these capabilities: 
+
+1. [Find Customer Service Insights resources containing customer data](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-discovery) in order to respond to GDPR requests from customers. 
+
+2. [Provide a copy of customer data](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-export) by using data export.
```
Comment (nenonix) [inline microsoft-365/compliance/gdpr-dsr-Dynamics365.md]:
```suggestion
2. [Provide a copy of customer data](https://docs.microsoft.com/dynamics365/ai/customer-service-insights/gdpr-export) using data export.
```

**Review by nenonix — CHANGES_REQUESTED** [review]
copy review

**Comment by shellyhaverkamp** [comment]
I realized that I needed to move the links to match the original article's organization, something I missed seeing the first time. I also fixed some inconsistent capitalization while I was at it. @mypil please see if someone on your side can review. Thank you!

**Review by BuckeyeGuyJFlo — APPROVED** [review]

**Pull request merged**